services/galley/src/Galley/API/Public.hs

```diff
@@ -177,7 +177,6 @@ sitemap = do
     -- nginz metrics dashboards easier.
     zauthUserId
       .&. capture "tid"
-      .&. accept "text" "csv"
```

Comment:
@Yserz you won't set the accept header in the csv download request, right?  or will you?

Reply:
With an anchor tag I can not modify any headers. The browser would add every mime-type to the header that is acceptable for the browser.

Reply:
Ok, let's remove it then, seems safer to me.  Thanks!